cmd/kubeadm/app/util/kubeconfig/kubeconfig_test.go

```diff
@@ -186,3 +188,143 @@ func TestWriteKubeconfigToDisk(t *testing.T) {
 		})
 	}
 }
+
+func TestGetCurrentAuthInfo(t *testing.T) {
```

Comment:
suggested minor improvements for the test names here.
please adjust as see fit.

Reply:
done, thanks